src/toolkit/step/OtkStepGenerateIIRDS.vue

```diff
@@ -302,8 +303,9 @@ export default {
                 */
                 if (object.source && object.source.data instanceof Blob && !!object.source.type) {
                     let filename = object.externalId || object.source.name || object.source.data.name;
+                    let sanitized = filename.replace(/[öüäßÖÜÄ#"'#\?\s]+/g, "-");
```

Comment:
Was spricht gegen escapen? Vermutlich wird der Regex nur ein Teil der invaliden Chars rausfischen.

**Beispiel:**
filename: **üöä.,.-°!!§$%&()(=)```´´+#__--..,,€=1=true.html**

regex: _rendition/-.,.-°!!§$%&amp;()(=)```´´+-__--..,,€=1=true.html_
escapeURI() => _rendition/%C3%BC%C3%B6%C3%A4.,.-%C2%B0!!%C2%A7$%25&()(=)%60%60%60%C2%B4%C2%B4+#__--..,,%E2%82%AC=1=true.html_

Reply:
ist halt etwas angenehmer zum Lesen und über die Dateinamenkonventionen werden die meisten Sonderzeichen schon abgefangen, aber ja, escapen wäre auch ne Möglichkeit

Reply:
Behoben in 28a5586